.github/workflows/ci.yml

```diff
@@ -10,9 +10,9 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.16'
-  GOLANGCI_VERSION: 'v1.31'
-  DOCKER_BUILDX_VERSION: 'v0.4.2'
+  GO_VERSION: '1.18'
+  GOLANGCI_VERSION: 'v1.47.1'
+  DOCKER_BUILDX_VERSION: 'v0.8.2'
```

Comment:
We copy and paste this CI flow around to a lot of places. Ideally we'd update them all but I don't want to volunteer you to update the world - maybe a tracking issue would be appropriate?

Reply:
I'll open a tracking issue in one of the internal repos and maybe write a one-off script to automate these PRs